src/kaltura-live-plugin.tsx

```diff
@@ -284,7 +284,7 @@ export class KalturaLivePlugin implements OnMediaUnload, OnPluginDestroy {
         this._initTimeout();
         switch (type) {
             case OfflineTypes.NoLongerLive:
-                if (!this._player.isDvr() || this._attachMedia) {
+                if (!this._player.isDvr() || this._mediaDetached || this.reloadMedia) {
```

Comment:
If we are already changing names - lets change reloadMedia to something that describes it and not the action that needs to be done.
For example - playerIsErrored 
That way it is clearer to read the condition